mPowerSDK/Dashboard/APHCorrelationsSelectorViewController.m

```diff
@@ -0,0 +1,203 @@
+    self.tableView.tableFooterView = [[UIView alloc] initWithFrame:CGRectZero];
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    [self.view addSubview:self.tableView];
+}
+
+- (void)moveUpTableView
+{
+    [UIView
+     animateWithDuration:0.5
+     animations:^{
+         self.tableView.frame = self.upFrame;
+     }];
+}
+
+- (void)moveDownTableView
+{
+    [UIView
+     animateWithDuration:0.5
+     animations:^{
+         self.tableView.frame = self.downFrame;
+     }
+     completion:^(BOOL finished) {
+         [self dismissViewControllerAnimated:NO completion:nil];
+    }];
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)__unused tableView
+{
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)__unused tableView numberOfRowsInSection:(NSInteger)__unused section
+{
+    return self.scoringObjects.count;
+}
+
+#pragma mark - UITableViewDelegate
+
+- (UIView *)tableView:(UITableView *)__unused tableView viewForHeaderInSection:(NSInteger)section
+{
+    NSString *cancelString = NSLocalizedStringWithDefaultValue(@"Cancel", @"APCAppCore", APHLocaleBundle(), @"Cancel", nil);
+    
+    UIButton *cancel = [UIButton buttonWithType:UIButtonTypeCustom];
+    cancel.frame = CGRectMake(self.tableView.frame.size.width - 100, 0, 100, 44.0);
+    [cancel setTitle:cancelString forState:UIControlStateNormal];
+    [cancel setTitleColor:[UIColor appTertiaryRedColor] forState:UIControlStateNormal];
+    [cancel addTarget:self action:@selector(dismissView) forControlEvents:UIControlEventTouchUpInside];
+    cancel.titleLabel.font = [UIFont fontWithName:@"HelveticaNeue" size:17];
+    
+    UITableViewHeaderFooterView *headerView = [[UITableViewHeaderFooterView alloc] init];
+    headerView.textLabel.text = NSLocalizedStringWithDefaultValue(@"Select Category", @"APCAppCore", APHLocaleBundle(), @"Select Category", nil);
+    [headerView addSubview:cancel];
+    
+    return headerView;
+}
+
+- (CGFloat)tableView:(UITableView *)__unused tableView heightForHeaderInSection:(NSInteger)__unused section
+{
+    return 44.0;
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"Cell" forIndexPath:indexPath];
+    
+    APCScoring *scoringObject = [self.scoringObjects objectAtIndex:indexPath.row];
+    cell.textLabel.text = scoringObject.caption;
+    [cell setAccessoryType:UITableViewCellAccessoryNone];
+    
+    if (scoringObject == self.selectedObject) {
+        cell.textLabel.textColor = self.view.tintColor;
+        [cell setAccessoryType:UITableViewCellAccessoryCheckmark];
+    } else {
+        cell.textLabel.textColor = [UIColor appSecondaryColor1];
+    }
+    
+    return cell;
+}
+
+- (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
```

Comment:
@evliu Not seeing update. Note: if this delegate method is defined, it will be called and is less performant. Can make a difference on an iPhone 4s (which we still support).